packages/react-server/src/ReactFlightServer.js

```diff
@@ -1279,6 +1284,11 @@ function renderAsyncFragment(
       REACT_FRAGMENT_TYPE,
       task.keyPath,
       {children},
+      // TODO: Pass debug stack and info?
+      __DEV__ ? null : undefined,
+      __DEV__ ? null : undefined,
+      // validated
+      __DEV__ ? 1 : undefined,
```

Comment:
Because RSC can encode `undefined` this is not equivalent to missing properties. This will show up in prod. Even if it was we should have consistent internal types on the server too that doesn't use extra fields.